resources/release-drafter.md

```diff
@@ -45,6 +42,24 @@ Ex: `0.1.4` -> `0.2.0`
 - If the integration is stable (e.g. `X.Y.Z` when `X > 0`) the major version increases.
 Ex: `2.1.4` -> `3.0.0`
 
+#### `enhancement`
+
+If the PR introduces a new feature or a new improvement: add the label `enhancement`.
+
+For integrations that are stable and so have a major release version (e.g. `1.x.x`, `2.x.x`), this will make the minor increase (e.g. `1.2.4` -> `1.3.0`).
+
+#### `bug`
+
+If the PR fixes bugs: add the label `bug`.
+
+A bug could be an unexpected raised error, but also a loss of performance, or any unexpected behavior of the integration.
```

Comment:
oops, I accepted, but I disagree finally. A raised error is not necessarily a bug. You can intend to raise an error in some situation. It will not be a bug